neo/Wallets/Wallet.py

```diff
@@ -165,42 +202,124 @@ def ChangePassword(self, password_old, password_new):
         self.SaveStoredData("MasterKey", AES.new(self._master_key, AES.MODE_CBC, self._iv))
 
     def ContainsKey(self, public_key):
+        """
+        Test if the wallet contains the supplied public key.
+
+        Args:
+            public_key (edcsa.Curve.point): a public key to test for its existance. i.e. KeyPair.PublicKey
+
+        Returns:
+            bool: True if exists, False otherwise.
+
+        """
         return self.ContainsKeyHash(Crypto.ToScriptHash(public_key.encode_point(True), unhex=False))
 
     def ContainsKeyHash(self, public_key_hash):
+        """
+        Test if the wallet contains the supplied public key hash in its key list.
+
+        Args:
+            public_key_hash (UInt160): a public key hash to test for its existance.
+
+        Returns:
+            bool: True if exists in wallet key list, False otherwise.
 
+        """
         return public_key_hash.ToBytes() in self._keys.keys()
 
     def ContainsAddress(self, script_hash):
+        """
+        Determine if the wallet contains the address.
+
+        Args:
+            script_hash (UInt160): a bytearray (len 20) representing the public key.
+
+        Returns:
+            bool: True, if the address is present in the wallet. False otherwise.
+
+        """
         return self.CheckAddressState(script_hash) >= AddressState.InWallet
 
     def ContainsAddressStr(self, address):
+        """
+        Determine if the wallet contains the address.
+
+        Args:
+            address (str): a string representing the public key.
+
+        Returns:
+            bool: True, if the address is present in the wallet. False otherwise.
+
+        """
+
```

Comment:
Also I'd say don't add empty lines after the docstring, start with the code straight away.